make.ps1

```diff
@@ -63,6 +63,7 @@ function create-smoke-tests() {
     Remove-Item Env:\LOG_LEVEL -ErrorAction SilentlyContinue
     cd tests/e2e/pact
     Remove-Item './pact' -Recurse -ErrorAction SilentlyContinue
+    npm run clear-cache
```

Comment:
What does this do and why do we need it?

Reply:
This runs npm target clear-cache in package.json, it runs jest --clearCache, this resolves an issue where if you're making heavy refactoring changes like deleting/renaming specs jest still attempts to run them even though they no longer exist